Source/Scene/Scene.js

```diff
@@ -945,6 +950,36 @@ define([
         },
 
         /**
+         * This property is for debugging only; it is not for production use.
+         * <p>
+         * When <code>true</code>, draws outlines to show the boundaries of the camera frustums
+         * </p>
+         *
+         * @type Boolean
+         *
+         * @default false
+         */
+        debugShowFrustumPlanes : {
+            get : function() {
+                return this._debugShowFrustumPlanes;
+            },
+            set : function(value) {
+                if (!value) {
```

Comment:
Here and below, do not call `destroy` and `new DebugCameraPrimitive` in the `set` function; it could result in WebGL calls being made outside of request animation frame.

Instead check if this flag changed in the render loop and then create or destroy.

For example, see `modelMatrix` in [DebugModelMatrixPrimitive](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Scene/DebugModelMatrixPrimitive.js).